ui/push-health/pushhealth.css

```diff
@@ -5,3 +5,13 @@
 .pointable {
   cursor: pointer;
 }
+
+.pre-wrap {
+  white-space: pre-wrap;
+  word-break: break-all;
```

Comment:
There's some bootstrap text utilities - `text-wrap` and `text-break` - that might help here? https://getbootstrap.com/docs/4.3/utilities/text/#text-wrapping-and-overflow

Reply:
Yeah, ``text-break`` would work instead of using ``break-all``.  Though I still wanted to have the ``pre-wrap`` part to format blocks like this.  But thanks! :)